### PR TITLE
Refactor data service resource in management api.

### PR DESCRIPTION
**Pull request by sachithKay**

### CODE REVIEW

File: components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java [hunk 1]
```diff
@@ -163,4 +171,33 @@ private ServiceMetaData getServiceMetaData(DataService dataService) throws Excep
...
+    private List<ResourceInfo> getResources(DataService dataService) {
+
+        Set<Resource.ResourceID> resourceIDS = dataService.getResourceIds();
+        List<ResourceInfo> resourceList = new ArrayList<>();
+        for (Resource.ResourceID id : resourceIDS) {
+            ResourceInfo resourceInfo = new ResourceInfo();
+            Resource resource = dataService.getResource(id);
+            resourceInfo.setResourcePath(id.getPath());
+            resourceInfo.setResourceMethod(id.getMethod());
+            resourceInfo.setResourceQuery(resource.getCallQuery().getQuery().getQueryId());
```
Comment (chanikag) [inline components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java:184]: Do we need query info in the dashboard view? It is kind of an internal information. Isn't it?

Comment (sachithKay) [reply components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java:184]: We need to show which query is related to the resource right? Please refer below.
Screen Shot 2020-06-25 at 9 46 47 AM

Comment (chanikag) [reply components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java:184]: It is an internal detail. We do not expose that from WSDL, etc. Instead we should expose query params.

Comment (sachithKay) [reply components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/DataServiceResource.java]: Modified as per the discussion. Added query params as well

**Pull request merged**